package.json

```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "@superfaceai/ast": "^1.0.0",
     "@superfaceai/cli": "^1.0.1",
-    "@superfaceai/one-sdk": "^1.0.0",
+    "@superfaceai/one-sdk": "^1.0.1-rc.1",
```

Comment:
Not sure if we want to have RCs in main. @freaz what do you think?

Reply:
I want to update to 1.0.1 but did not want to block review with it. There is task for it in description.

Reply:
one-sdk updated to v1.0.1